bin/phctl.rb

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "thor"
+require "pry"
+require "services"
+require "loader/file_loader"
+
+Services.mongo!
+
+module PHCTL
+  class Load < Thor
+    desc "commitments FILENAME", "Add shared print commitments"
+    def commitments(filename)
+      require "loader/shared_print_loader"
+      Services.logger.info "Loading Shared Print Commitments: #{filename}"
+      Loader::FileLoader.new(batch_loader: Loader::SharedPrintLoader.new).load(filename) 
+    end
+
+    desc "ht_items FILENAME", "Add HT Items"
+    def ht_items(filename)
+      require "loader/ht_item_loader"
+      Services.logger.info "Updating HT Items."
+      Loader::FileLoader.new(batch_loader: Loader::HtItemLoader.new).load(filename)
+    end
+
+    desc "concordance DATE", "Load concordance deltas for the given date"
+    def concordance(date)
+      require "ocn_concordance_diffs"
+      OCNConcordanceDiffs.new(Date.parse(date)).load
+    end
+  end
+
+  class Concordance < Thor
+  
+    desc "validate INFILE OUTFILE",  "Validate a concordance file"
+    def validate(infile, outfile)
+      require_relative "concordance_validation/validate.rb"
+      main(infile, outfile)
+    end
+
+    desc "validate-and-delta", "Validates and computes deltas in default concordance directory"
+    def validate_and_delta
+      require_relative "concordance_validation/validate_and_delta.rb"
+      main
+    end 
+  end
+
+
+  class Report < Thor
+    desc "costreport ORG", "Run a cost report"
+    def costreport(org=nil)
+      require_relative "reports/compile_cost_reports.rb"
```

Comment:
We will want to get these `main` functions out of the global namespace so we don't need to do the `require_relative` in the block; otherwise, this is looking pretty good.